config/annotations.js

```diff
@@ -26,7 +26,8 @@ const RESERVED_JOB_ANNOTATIONS = [
     'screwdriver.cd/manualStartEnabled',
     'screwdriver.cd/blockedBySameJob',
     'screwdriver.cd/blockedBySameJobWaitTime',
-    'screwdriver.cd/jobDisabledByDefault'
+    'screwdriver.cd/jobDisabledByDefault',
+    'screwdriver.cd/virtual'
```

Comment:
Would the name `virtualTrigger` make more sense?

Reply:
I think virtualTrigger would be more clear

Reply:
Done. Changed to `virtualTrigger`.

Reply:
Intention is that the job is a virtual job. Where does trigger come into picture ?